spec/src/main/asciidoc/interoperability.asciidoc

```diff
@@ -419,6 +419,16 @@ public class RolesEndpoint {
 
 ```
 
+If there is no JWT in the request, an empty `JsonWebToken` is injected, which means all method calls to this token
+return `null`. Note that MP JWT will still perform Authentication and Authorization if the endpoint requires these
+verifications. Effectively, the injected empty token  is only visible on unauthenticated and unauthorized endpoints.
+
+If a JWT is sent to an endpoint that does not require Authentication and/or Authorization then it still must be
+verified before it can be accessed via JsonWebToken interface`
```

Comment:
@radcortez please replace ``` with `.` at the end. And wrap `JsonWebToken` instead.
By the way, if you meant `CDI.getInstance` when asking what happens when no injection point is available then it was a very valid question - to me it is all the same I guess, `CDI.getInstance` implies getting in touch with the producer I guess, but I think it was useful to touch on it, and I hope the text leaves no ambiguity now, thanks

Reply:
Ops sorry. Wrong copy paste :)

No worries, is all semantics. The issue is that someone might interpret that validation would only occur if you have an injection point.